src/Behat/Symfony2Extension/ServiceContainer/Symfony2Extension.php

```diff
@@ -72,8 +72,8 @@ public function configure(ArrayNodeDefinition $builder)
                     ->addDefaultsIfNotSet()
                     ->children()
                         ->scalarNode('bootstrap')->defaultValue(self::DEFAULT_KERNEL_BOOTSTRAP)->end()
-                        ->scalarNode('path')->defaultValue('app/AppKernel.php')->end()
-                        ->scalarNode('class')->defaultValue('AppKernel')->end()
+                        ->scalarNode('path')->defaultValue(class_exists('App\Kernel') ? 'src/AppKernel.php' : 'app/AppKernel.php')->end()
+                        ->scalarNode('class')->defaultValue(class_exists('App\Kernel') ? 'App\Kernel' : 'AppKernel')->end()
```

Comment:
Should I create a separate `testapp` to properly test this? Or maybe change the existing one? Or not bother?